<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>578</width>
    <height>539</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>80</y>
     <width>32</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>alpha</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="particleNum">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>20</y>
     <width>167</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>50</y>
     <width>32</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>scale</string>
   </property>
  </widget>
  <widget class="QSlider" name="particleScale">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>50</y>
     <width>167</width>
     <height>15</height>
    </rect>
   </property>
   <property name="maximum">
    <number>1000</number>
   </property>
   <property name="value">
    <number>100</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>20</y>
     <width>74</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>numParticles</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="attractor_b">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>150</y>
     <width>67</width>
     <height>21</height>
    </rect>
   </property>
   <property name="singleStep">
    <double>0.010000000000000</double>
   </property>
   <property name="value">
    <double>1.800000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="attractor_c">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>190</y>
     <width>67</width>
     <height>21</height>
    </rect>
   </property>
   <property name="singleStep">
    <double>0.010000000000000</double>
   </property>
   <property name="value">
    <double>0.710000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="attractor_a">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>120</y>
     <width>67</width>
     <height>21</height>
    </rect>
   </property>
   <property name="singleStep">
    <double>0.010000000000000</double>
   </property>
   <property name="value">
    <double>1.000000000000000</double>
   </property>
  </widget>
  <widget class="QSlider" name="particleAlpha">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>80</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="maximum">
    <number>1000</number>
   </property>
   <property name="value">
    <number>50</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="attractor_d">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>230</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="singleStep">
    <double>0.010000000000000</double>
   </property>
   <property name="value">
    <double>1.510000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>300</y>
     <width>56</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>scale</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>330</y>
     <width>56</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>alpha</string>
   </property>
  </widget>
  <widget class="QSlider" name="billboardScale">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>300</y>
     <width>160</width>
     <height>16</height>
    </rect>
   </property>
   <property name="maximum">
    <number>1000</number>
   </property>
   <property name="value">
    <number>1000</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="billboardAlpha">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>330</y>
     <width>160</width>
     <height>16</height>
    </rect>
   </property>
   <property name="maximum">
    <number>1000</number>
   </property>
   <property name="value">
    <number>100</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>380</y>
     <width>56</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>frequency</string>
   </property>
  </widget>
  <widget class="QSlider" name="frequency">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>380</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="maximum">
    <number>1000</number>
   </property>
   <property name="value">
    <number>50</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>410</y>
     <width>56</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>octaves</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>440</y>
     <width>56</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>lacunarity</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>480</y>
     <width>56</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>gain</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="octaves">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>410</y>
     <width>53</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>8</number>
   </property>
  </widget>
  <widget class="QSlider" name="lacunarity">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>440</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="maximum">
    <number>1000</number>
   </property>
   <property name="value">
    <number>50</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="gain">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>480</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="maximum">
    <number>1000</number>
   </property>
   <property name="value">
    <number>500</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
